src/OrchardCore/OrchardCore.Mvc.Core/Startup.cs

```diff
@@ -55,6 +55,10 @@ public override void ConfigureServices(IServiceCollection services)
             // Add a route endpoint selector policy.
             services.AddSingleton<MatcherPolicy, FormValueRequiredMatcherPolicy>();
 
+            // There are some issues when using the default formatters based on
```

Comment:
What kind of issues?

Reply:
I will repro the issue and give you more details

Reply:
For example, when running a graphQL unit test, in `SiteContext.cs` we have this.

            // Note: 'createResult' is an 'HttpResponseMessage'
            var content = await createResult.Content.ReadAsStringAsync();

            var url = new Uri(content.Trim('"'));

Without `AddNewtonsoftJson()` it fails when creating the `Uri`, here the `content` variable value is.

    "\"http:\\u002f\\u002flocalhost\\u002f39e1684e3e4a420386374fd388010dc2?token=...""

With `AddNewtonsoftJson()` it doesn't fails, here the `content` variable value is.

    "\"http://localhost/320e5a211fda47799ea23e6aea32b2c1?token=...""

Reply:
But there is a new `System.Text.Json.JsonSerializerOptions` class, i will take a look.

Reply:
Anyway, if i "fix" the above issue by enescaping the string, then there is another issue when doing.

    await Client.PostAsJsonAsync("api/content" + (draft ? "?draft=true" : ""), contentItem);

saying this, knowing that the `ContentItem.Content` property is a `JObject`.

    System.NotSupportedException : The collection type 'Newtonsoft.Json.Linq.JObject' on
    'OrchardCore.ContentManagement.ContentItem.Content' is not supported.


This at

    at System.Text.Json.JsonClassInfo.GetElementType(Type propertyType, Type parentType, MemberInfo memberInfo)
    at System.Text.Json.JsonClassInfo.CreateProperty(Type declaredPropertyType, Type runtimePropertyType, PropertyInfo propertyInfo, Type parentClassType, JsonSerializerOptions options)
    at System.Text.Json.JsonClassInfo.CreatePolymorphicProperty(JsonPropertyInfo property, Type runtimePropertyType, JsonSerializerOptions options)
    at System.Text.Json.JsonSerializer.GetRuntimePropertyInfo(Object value, JsonClassInfo jsonClassInfo, JsonPropertyInfo& jsonPropertyInfo, JsonSerializerOptions options)
    at System.Text.Json.JsonSerializer.HandleObject(JsonSerializerOptions options, Utf8JsonWriter writer, WriteStack& state)
    at System.Text.Json.JsonSerializer.WriteObject(JsonSerializerOptions options, Utf8JsonWriter writer, WriteStack& state)
    at System.Text.Json.JsonSerializer.Write(Utf8JsonWriter writer, Int32 flushThreshold, JsonSerializerOptions options, WriteStack& state)
    at System.Text.Json.JsonSerializer.WriteAsyncCore(Stream utf8Json, Object value, Type type, JsonSerializerOptions options, CancellationToken cancellationToken)
    at Microsoft.AspNetCore.Mvc.Formatters.SystemTextJsonOutputFormatter.WriteResponseBodyAsync(OutputFormatterWriteContext context, Encoding selectedEncoding)

Reply:
JObject not being serializable is expected and explains why we need Json.net. The first issue though needs to be investigated to see if there is an issue with System.Text.Json.

Reply:
Okay, i will let you know if i find something